boot/zephyr/Kconfig

```diff
@@ -43,6 +44,29 @@ config BOOT_USE_TINYCRYPT
 	help
 	  Use TinyCrypt for crypto primitives.
 
+config BOOT_USE_CC310
+	bool
+	# Hidden option
+	default n
+	# When building for ECDSA, we use our own copy of mbedTLS, so the
+	# Zephyr one must not be enabled or the MBEDTLS_CONFIG_FILE macros
+	# will collide.
+	depends on ! MBEDTLS
+	help
+	  Use cc310 for crypto primitives.
+
+config BOOT_USE_NRF_CC310_BL
+	bool
+	default n
+
+config NRFXLIB_CRYPTO
```

Comment:
I don't think this is needed for downstream as we get the Kconfig from nrfxlib? through the west manifest in nrf

Reply:
@SebastianBoe ^^

Reply:
I'll remove it and add a west manifest to MCUBoot upstream

Reply:
can you do it quickly here as [nrf fromlist] commit and send path to upsterem as well?

Reply:
Sure I'll do that